tracing-journald/src/lib.rs

```diff
@@ -109,6 +114,48 @@ impl Subscriber {
         self.field_prefix = x;
         self
     }
+
+    #[cfg(not(unix))]
+    fn send_payload(&self, _opayload: &[u8]) -> std::io::Result<()> {
```

Comment:
nit: i believe `std::io` is already imported in this scope:
```suggestion
    fn send_payload(&self, _opayload: &[u8]) -> io::Result<()> {
```